library/src/main/java/com/avast/android/dialogs/core/BaseDialogFragment.java

```diff
@@ -26,12 +26,14 @@
 import android.content.res.TypedArray;
 import android.graphics.Typeface;
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.annotation.StyleRes;
-import android.support.v4.app.DialogFragment;
-import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentManager;
-import android.support.v4.app.FragmentTransaction;
+
```

Comment:
Please keep this order of imports 
- java*
- empty line
- android.*
- empty line
- androidx.*
- empty line
- other

Reply:
Done.